pybikes/bixi.py

```diff
@@ -1,211 +0,0 @@
-            "city":"",
-            "postalCode":"",
-            "location":"",
-            "altitude":"",
-            "testStation":false,
-            "lastCommunicationTime":null,
-            "landMark":""
-          }
-        '''
-        station = BixiStation()
-        if data['statusValue'] == 'Planned' or data['testStation']:
-            raise exceptions.StationPlannedException()
-
-        station.name      = "%s - %s" % (data['id'], data['stationName'])
-        station.longitude = float(data['longitude'])
-        station.latitude  = float(data['latitude'])
-        station.bikes     = int(data['availableBikes'])
-        station.free      = int(data['availableDocks'])
-
-        station.extra = {
-            'uid': int(data['id']),
-            'statusValue': data['statusValue'],
-            'statusKey': data['statusKey'],
-            'stAddress1': data['stAddress1'],
-            'stAddress2': data['stAddress2'],
-            'city': data['city'],
-            'postalCode': data['postalCode'],
-            'location': data['location'],
-            'altitude': data['altitude'],
-            'testStation': data['testStation'],
-            'lastCommunicationTime': data['lastCommunicationTime'],
-            'landMark': data['landMark'],
-            'totalDocks': data['totalDocks']
-        }
-
-        return station
-
-    @staticmethod
-    def from_json_xml(data):
-        """ json marker object translated from xml
-        {
-            "id": "2",
-            "name": "Docklands Drive - Docklands",
-            "terminalName": "60000",
-            "lastCommWithServer": "1375644471147",
-            "lat": "-37.814022",
-            "long": "144.939521",
-            "installed": "true",
-            "locked": "false",
-            "installDate": "1313724600000",
-            "removalDate": {  },
-            "temporary": "false",
-            "public": "true",
-            "nbBikes": "15",
-            "nbEmptyDocks": "8",
-            "latestUpdateTime": "1375592453128"
-        }
-        """
-
-        station = BixiStation()
-
-        station.name = "%s - %s" % (data['terminalName'], data['name'])
-        station.latitude = float(data['lat'])
-        station.longitude = float(data['long'])
-        station.bikes = int(data['nbBikes'])
-        station.free = int(data['nbEmptyDocks'])
-
-        station.extra = {
-            'uid': int(data['id']),
-            'name': data['name'],
-            'terminalName': data['terminalName'],
-            'lastCommWithServer': data['lastCommWithServer'],
-            'installed': utils.str2bool(data['installed']),
-            'locked': utils.str2bool(data['locked']),
-            'installDate': data['installDate'],
-            'removalDate': data['removalDate'],
-            'temporary': utils.str2bool(data['temporary']),
-            'public': utils.str2bool(data['public']),
-            'latestUpdateTime': data['latestUpdateTime']
-        }
-        return station
```

Comment:
This file cannot be deleted yet. Other systems (these are present on [`pybikes/data/bixi.json`](https://github.com/f8full/pybikes/blob/7f043af9940802397d62ad584dedad772df0964b/pybikes/data/bixi.json)) are using it.

Reply:
Ooopsy
I learnt my lesson
https://github.com/f8full/pybikes/commit/7c3cdf9aecf22975805a7d2570e4f70edbb8dfc1

someday I may write a parser for this file that would feed pybikes
https://github.com/NABSA/gbfs/blob/master/systems.csv

cheers,